.github/workflows/slack-pr-bot.yml

```
@@ -0,0 +1,552 @@
+
+jobs:
+  notify-slack:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_USER_MAP: '{"iOdiO89":"U0AKB7N0C66","kanghaeun":"U0AKA5ZNYEA","soyeong0115":"U0AK4SC7C2X","ychany":"U0AK6MZM9RR","sevineleven":"U0AK6U2J62F"}'
+
+    steps:
+      - name: Extract PR data
+        id: pr
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)